### PR TITLE
Function to download an image set complete

### PR DESCRIPTION
**Pull request by jondionowens**

### CODE REVIEW

**Comment by allenhsieh** [comment]
Looks really good! One suggestion, do you need the images in your database/downloads folder on github? I think you can add that folder to your .gitignore (since they'll be downloaded by the person cloning/building/running)

File: .gitignore [hunk 1]
```diff
@@ -24,3 +24,6 @@ coverage
 build/
 dist/
 
+
+database/downloads
```
Comment (allenhsieh) [inline .gitignore:28]: ah! i see you ARE ignoring here, what the heck, why are all these images still uploading?

File: public/index.html [hunk 2]
```diff
@@ -9,7 +9,7 @@
   <div id="photo-service"></div>
   <script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
   <script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.development.js"></script>
-  <script src="http://localhost:3000/app.js"></script>
+  <script src="http://localhost:3002/app.js"></script>
   <script>
     ReactDOM.render(
```
Comment (allenhsieh) [inline public/index.html:14]: So what we found out is you don't need to do it this way! the example folder is kind of bad! you remember how in the past, in your client's index.js file there was something like this? 
```
import React from 'react';
import ReactDOM from 'react-dom';
import DetailsApp from './components/DetailsApp.jsx';
window.DetailsApp = DetailsApp;

ReactDOM.render(<DetailsApp />, document.getElementById('Details'));
```

This allows us to wrap up react/reactDOM and the reactDOM.render all in the bundle

**Comment by jondionowens** [comment]
@allenhsieh Updated to remove the downloads folder and images + some further commits

**Review by allenhsieh — APPROVED** [review]

**Pull request merged**